concepticondata/conceptlists/Zhang-2019-308.tsv

```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-35	35	the cooked rice	cooked rice	1191	COOKED RICE	
+Zhang-2019-308-36	36	the cow	cow	1007	COW	
+Zhang-2019-308-37	37	to cry (of birds or animals)	cry (of birds or animals)	1839	CRY	
+Zhang-2019-308-38	38	the dark side of the mountain	dark side of the mountain			too specific to map
+Zhang-2019-308-39	39	the deceased mother	deceased mother			too specific to map
+Zhang-2019-308-40	40	the dream	dream	2374	DREAM	
+Zhang-2019-308-41	41	the drop (classifier)	drop (classifier)	2087	DROP (CLASSIFIER)	
+Zhang-2019-308-42	42	be dusk, dark	dusk, dark	2297	DUSK	
+Zhang-2019-308-43	43	the ear	ear	1247	EAR	
+Zhang-2019-308-44	44	the earthworm	earthworm	2350	EARTHWORM	
+Zhang-2019-308-45	45	eight	eight	1705	EIGHT	
+Zhang-2019-308-46	46	the elbow	elbow	981	ELBOW	
+Zhang-2019-308-47	47	be eldest, great	eldest, great			concept only existent in Old Chinese
+Zhang-2019-308-48	48	be evil	evil	3764	BAD OR EVIL	
+Zhang-2019-308-49	49	the excrement	excrement	675	FAECES (EXCREMENT)	
+Zhang-2019-308-50	50	the eye	eye	1248	EYE	
+Zhang-2019-308-51	51	the eyebrow	eyebrow	1173	EYEBROW	
+Zhang-2019-308-52	52	the face	face	1560	FACE	
+Zhang-2019-308-53	53	the farming	farming			
+Zhang-2019-308-54	54	the farmland	farmland	201	FARM	
+Zhang-2019-308-55	55	the father	father	1217	FATHER	
+Zhang-2019-308-56	56	the father's brother	father's brother	1985	PATERNAL UNCLE (FATHER'S BROTHER)	
+Zhang-2019-308-57	57	the father's sister's child	father's sister's child			
+Zhang-2019-308-58	58	the field	field	212	FIELD	
+Zhang-2019-308-59	59	the fire	fire	221	FIRE	
+Zhang-2019-308-60	60	the firewood	firewood	10	FIREWOOD	
+Zhang-2019-308-61	61	five	five	493	FIVE	
+Zhang-2019-308-62	62	to float	float	1574	FLOAT	
+Zhang-2019-308-63	63	the fly	fly (insect)	1504	FLY (INSECT)	
+Zhang-2019-308-64	64	be fresh	fresh	243	FRESH	
+Zhang-2019-308-65	65	the friend	friend	1325	FRIEND	
+Zhang-2019-308-66	66	the fur garment	fur garment			
+Zhang-2019-308-67	67	the garment	garment	1895	CLOTHES	
+Zhang-2019-308-68	68	the grandchild, nephew	grandchild, nephew	1617	GRANDCHILD	
+Zhang-2019-308-69	69	the grandmother	grandmother	1496	GRANDMOTHER	
+Zhang-2019-308-70	70	the hair	hair	1040	HAIR	
+Zhang-2019-308-71	71	the hat	hat	771	HAT	
+Zhang-2019-308-72	72	the head	head	1256	HEAD	
+Zhang-2019-308-73	73	the heart	heart	1223	HEART	
+Zhang-2019-308-74	74	the hide	hide	3326	LEATHER OR HIDE	
+Zhang-2019-308-75	75	the hole	hole	1667	HOLE	
+Zhang-2019-308-76	76	the honey	honey	942	HONEY	
+Zhang-2019-308-77	77	the horn	horn	1393	HORN (ANATOMY)	
+Zhang-2019-308-78	78	the horse	horse	615	HORSE	
+Zhang-2019-308-79	79	hundred	hundred	1634	HUNDRED	
+Zhang-2019-308-80	80	the husband	husband	1200	HUSBAND	
+Zhang-2019-308-81	81	the ice	ice	617	ICE	
+Zhang-2019-308-82	82	the intestine	intestine	1991	INTESTINES	
+Zhang-2019-308-83	83	the internal organs	internal organs	1334	GUTS	
+Zhang-2019-308-84	84	the intestine	intestine	1991	INTESTINES	
+Zhang-2019-308-85	85	the jaw, chin	jaw, chin	798	JAW	
```

Comment:
In Nagano-2013-1256-37 'jaw, chin' is mapped to CHIN but I don't know what lexemes are used here and in Nagano's list.

Reply:
I agree, I think CHIN is a better option, but @laiyunfan could you have a look?

Reply:
Frankly, I don't know the exact differences between chin and jaw in this case. But if Nagano has CHIN, let it be CHIN.